lib/routes/auth-router.js

```diff
@@ -6,48 +7,54 @@ const router = express.Router();
 const Users = require('../models/users-model.js');
 const users = new Users();
 
-// TODO JSDoc Comment
+/**
+ * Cretes new user document in mongo db
+ * @param  {} req
+ * @param  {} res
+ * @param  {} next 
+ */
 const create = async (req, res, next) => {
   let user = await users.create(req.body);
   req.user = user && user._id ? user : null;
 
   next();
 };
+/**
+ *  authenticate user
+ * @param  {} req
+ * @param  {} res
+ * @param  {} next
+ */
```

Comment:
Try to be more specific on declaring the types like: `@param {object} req`, and `@param {function} next`